README.md

```diff
@@ -8,34 +8,85 @@
   </a>
 </p>
 
-The Embrace Android SDK gives you performance and stability insights into the user experience of your mobile apps.
-
 [![codecov](https://codecov.io/gh/embrace-io/embrace-android-sdk/graph/badge.svg?token=4kNC8ceoVB)](https://codecov.io/gh/embrace-io/embrace-android-sdk)
 [![android api](https://img.shields.io/badge/Android_API-21-green.svg "Android min API 21")](https://dash.embrace.io/signup/)
 [![build](https://img.shields.io/github/actions/workflow/status/embrace-io/embrace-android-sdk/ci-gradle.yml)](https://github.com/embrace-io/embrace-android-sdk/actions)
+[![latest version](https://shields.io/github/v/release/embrace-io/embrace-android-sdk)](https://shields.io/github/v/release/embrace-io/embrace-android-sdk)
+
+# About
+The Embrace Android SDK builds on top of [OpenTelemetry](https://opentelemetry.io) to capture performance data for 
+Android apps, enabling full-stack observability of your system by connecting mobile and backend telemetry in a seamless way.
+
+Telemetry recorded through this SDK can consumed on the Embrace platform for Embrace customers, but it can also be used by those who are
+not Embrace customers to export collected data directly to any OTel Collector. In effect, this SDK is an alternative to [opentelemetry-android](https://github.com/open-telemetry/opentelemetry-android) or using the 
+[OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java) directly for Android apps that want to leverage the 
+OpenTelemetry ecosystem for observability. Currently, only Spans and Logs are supported, but other signals will be added in the future.
 
 # Getting Started
+## Non-Embrace Users
+### Android Project Setup
+
+1. In your app's Gradle file, add a dependency to the latest version of the Embrace Swazzler Gradle plugin: `io.embrace:embrace-swazzler:<version>`.
```

Comment:
Should contain the same code snippet as the integration docs IMO

Reply:
Sounds good. I'll try to do that, if not now then later.